deployment-sshd.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    name: sshd
+  name: sshd
```

Comment:
If you like the image name change idea, don't forget to change the name here and the rest of the file